src/class-model.ts

```diff
@@ -303,6 +306,10 @@ export function register<Instance, Klass extends { new (...args: any[]): Instanc
         return {
           afterCreate() {
             for (const view of klass.snapshottedViews) {
+              if (view.options.shouldTrackChanges && !view.options.shouldTrackChanges(self)) {
+                continue;
+              }
```

Comment:
I wonder if we could have some way of specifying a global default?

In Gadget's case, expecting engineers to remember to pass this option when they introduce a new snapshotted view is likely error-prone. I'm thinking something like:

```
setDefaultShouldTrackChanges(() => ...);
```

Reply:
agreed this seems wise